package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@hawktracer/client",
+  "version": "1.0.0",
+  "description": "A library that obtains profile data generated by HawkTracer",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "repository": "https://github.com/amzn/hawktracer.git",
+  "author": "Hojin Ghim <hojin@amazon.com>",
+  "license": "MIT",
+  "scripts": {
+    "test": "jest",
+    "prepare": "tsc",
+    "install": "cmake-js compile --target nodejs_bindings_client --CDENABLE_CLIENT=YES --CDENABLE_NODEJS_BINDINGS=YES --CDBUILD_STATIC_LIB=YES"
```

Comment:
It's not required.
But without it, we get these files after build (in mac).
```
❯ ls build/Debug
hawk_tracer_client.node  libhawktracer.dylib  libhawktracer_client_utils.dylib  libhawktracer_parser.dylib
```
And this file separation is purely implementation-specific. If these are published somewhere, it might lead to problems when we makes changes on that file separation.

By statically building them, we get only one file "hawk_tracer_client.node" which is simpler to maintain.